### PR TITLE
Fix for 'missing `else` statement in ReactDOMServerFormatConfig.js'

### PR DESCRIPTION
**Pull request by amensum**
## Summary

Fix for 'missing `else` statement in ReactDOMServerFormatConfig.js' (#22309).

## How did you test this change?

Tested with `yarn test`.

### CODE REVIEW

File: packages/react-dom/src/server/ReactDOMServerFormatConfig.js [hunk 1]
```diff
@@ -975,7 +975,6 @@ function pushStartTextArea(
         children.length <= 1,
         '<textarea> can only have at most one child.',
       );
```
Comment (justingrant) [inline packages/react-dom/src/server/ReactDOMServerFormatConfig.js:977]:
```suggestion
      );
      // Fall through. `'' + children` is same as `'' + children[0]` for 1-element arrays.
```
This behavior is weird enough that it probably deserves a comment explaining what's going on.

Comment (amensum) [reply packages/react-dom/src/server/ReactDOMServerFormatConfig.js:977]: Added comment

**Review by akgupta0777 — APPROVED** [review]

**Comment by justingrant** [comment]
Please rebase. #22064 was just merged which created a merge conflict.

**Comment by amensum** [comment]
@justingrant Check pls

**Comment by justingrant** [comment]
Hi @amensum - Normally PRs use a separate branch, not `main`. Also, merge commits are usually not present in PRs-- rebasing and force-pushing is usually better. 

Given that there's already another PR with the same change, I'd suggest closing this PR and we can continue the fix over at #22431.

**Pull request closed without merging**